### ISSUE TITLE
Exception when running a TensorFlow frozen graph model

### ISSUE THREAD

**Issue by b005t3r** [deeplearning4j/deeplearning4j#9171]
I get the following exception, when I try to run a TF frozen graph model:

```
java.lang.RuntimeException: offset + length is higher than original length
java.lang.RuntimeException: java.lang.RuntimeException: offset + length is higher than original length
	at org.nd4j.nativeblas.OpaqueDataBuffer.createView(OpaqueDataBuffer.java:150)
	at org.nd4j.linalg.cpu.nativecpu.buffer.BaseCpuDataBuffer.<init>(BaseCpuDataBuffer.java:168)
	at org.nd4j.linalg.cpu.nativecpu.buffer.FloatBuffer.<init>(FloatBuffer.java:85)
	at org.nd4j.linalg.cpu.nativecpu.buffer.DefaultDataBufferFactory.create(DefaultDataBufferFactory.java:68)
	at org.nd4j.linalg.factory.Nd4j.createBuffer(Nd4j.java:1061)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.<init>(BaseNDArray.java:194)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.<init>(BaseNDArray.java:189)
	at org.nd4j.linalg.cpu.nativecpu.NDArray.<init>(NDArray.java:91)
	at org.nd4j.linalg.cpu.nativecpu.CpuNDArrayFactory.create(CpuNDArrayFactory.java:420)
	at org.nd4j.linalg.factory.Nd4j.create(Nd4j.java:4032)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.create(BaseNDArray.java:2002)
	at org.nd4j.linalg.api.ndarray.BaseNDArray.get(BaseNDArray.java:4284)
	at org.nd4j.autodiff.samediff.internal.InferenceSession.getOutputsHelperTensorArrayOps(InferenceSession.java:709)
	at org.nd4j.autodiff.samediff.internal.InferenceSession.doExec(InferenceSession.java:462)
	at org.nd4j.autodiff.samediff.internal.InferenceSession.getOutputs(InferenceSession.java:222)
	at org.nd4j.autodiff.samediff.internal.InferenceSession.getOutputs(InferenceSession.java:68)
	at org.nd4j.autodiff.samediff.internal.AbstractSession.output(AbstractSession.java:391)
	at org.nd4j.autodiff.samediff.SameDiff.directExecHelper(SameDiff.java:2579)
	at org.nd4j.autodiff.samediff.SameDiff.batchOutputHelper(SameDiff.java:2547)
	at org.nd4j.autodiff.samediff.SameDiff.output(SameDiff.java:2522)
	at org.nd4j.autodiff.samediff.config.BatchOutputConfig.output(BatchOutputConfig.java:143)
...
Caused by: java.lang.RuntimeException: offset + length is higher than original length
	at org.nd4j.nativeblas.Nd4jCpu.dbCreateView(Native Method)
	at org.nd4j.nativeblas.OpaqueDataBuffer.createView(OpaqueDataBuffer.java:133)
```

```java
        byte[] data = new byte[inputTensorSize * inputTensorSize * 3];
        // initilize data with image data
        INDArray input = Nd4j.create(data, new long[]{1, inputTensorSize, inputTensorSize, 3}, DataType.UINT8);

        SameDiff sd = SameDiff.importFrozenTF(modelFile);
        Map<String, INDArray> result = sd.batchOutput()
                .input("image_tensor", input)
                .output("detection_boxes", "detection_classes", "detection_scores")
                .output();

```

Expected behavior: it works and I get output INDArrays.
Encountered behavior: it crashes with an exception.
Deeplearning4j version: 1.0.0-beta7
Platform information: macOS Big Sur (11.1 (20C69)).

**Comment by agibsonccc** [comment]
@b005t3r mind posting the TF file for me to look at? I may have already fixed this issue in master.

**Comment by b005t3r** [comment]
@agibsonccc sure, here's the model I tried: https://wetransfer.com/downloads/1e92dddd5696a50ffe7a0614bf30f61220210201141718/4221182af24f33ebd738ef6db1d416b820210201141731/029be8

Let me know if it works on your side and if there's anything I can do to make it work on my end :)

**Comment by b005t3r** [comment]
@agibsonccc sorry for being so inpatient, but have you tried the model yet?

**Comment by agibsonccc** [comment]
@b005t3r  no I haven't yet. We're in the middle of released QA. I'll do this when I get time. I plan on testing this against the current release. I'll try to get to this tomorrow. It shouldn't be too much for me to just try to load the model yet. If you need a guaranteed SLA, please DM me on the forums: https://community.konduit.ai/

**Comment by b005t3r** [comment]
@agibsonccc no problem, just please let me know here once you checked it and, if it works, what are my options to make it work on my end as well. Thanks!

**Comment by agibsonccc** [comment]
@b005t3r  when/if there's a workaround it will be problem specific. If something can work for beta7 I'll suggest that otherwise you might have to build from source till we get the release out.

**Comment by b005t3r** [comment]
@agibsonccc thanks, hopefully building from source won't be necessary :)

**Comment by agibsonccc** [comment]
@b005t3r JFYI I didn't forget about you, I cracked this open tonight but need to look at this a bit tomorrow morning my time (asia)

**Comment by b005t3r** [comment]
@agibsonccc great, good luck :) let me know how it went :)

**Comment by agibsonccc** [comment]
@b005t3r I managed to reproduce this on a few different models.  I'll make sure to get this fixed before release. Thanks for the bug report!

**Comment by b005t3r** [comment]
@agibsonccc thank you. When do you plan to release the new version? Matter of days, or weeks/months?

**Comment by agibsonccc** [comment]
Weeks at the latest but I'll let you know when I get the fix merged in to master. We're preparing for a 1.0 release release candidate so there's a bit of a higher bar for this.

**Comment by b005t3r** [comment]
@agibsonccc great, thank you

**Comment by b005t3r** [comment]
@agibsonccc, any news about the release? Do you think you will be able to release it this month?

**Comment by agibsonccc** [comment]
@b005t3r sorry looking in to this now. We're about to do a release so I'm processing these issues in batches. Before that I was focused on build infra. If you want a guaranteed SLA, you are welcome to sign a commercial agreement with us. 
I appreciate your patience as I look in to this. Sorry it took so long, there was a bit of a transition period for the infra. Running workloads on lots of different platforms and making sure it's well tested takes a bit :)

Edit: actually some other folks ran in to this. I believe we should have fixed this already. Do you mind trying snapshots if you have time? It might already work for you.

**Comment by b005t3r** [comment]
@agibsonccc thanks for the info, I think I should be able to test your snapshots next week. What should I get and where can I get it? Are they available via maven?

**Comment by b005t3r** [comment]
@agibsonccc I wanted to try out the snapshots, but it looks like there's no macOS build of 1.0.0-SNAPSHOT and 0.9.2-SNAPSHOT is 3 years old, so probably not what you wanted me to test. Is there a reason why there's no macOS build for this version?

**Comment by agibsonccc** [comment]
@b005t3r  reuploading snapshots for mac. Let me look at the other ones. The snapshots tend to time out after a few days so they need to be refreshed. Luckily we're working on final testing for the M1 release coming up here as well. Should be this week depending on what comes up in testing.

**Comment by b005t3r** [comment]
Sounds great! I'll test things out once the macOS builds are uploaded.

BTW, this is the repo I tried for snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/nd4j/nd4j-native/1.0.0-SNAPSHOT/

Is it the correct one?

**Comment by agibsonccc** [comment]
Yes it is thanks!

**Comment by b005t3r** [comment]
@agibsonccc Just wanted to make sure - you're still uploading the macOS snapshot builds?

**Comment by agibsonccc** [comment]
@b005t3r  I'm having it upload snapshots for osx now, it might take a bit.  Ping me tomorrow (sorry I live in asia so it's late here) if there's any issues.

**Comment by b005t3r** [comment]
@agibsonccc I'm afraid something went wrong with uploading the macOS snapshots, they are still not available:

```
 The following artifacts could not be resolved: org.nd4j:nd4j-native-preset:jar:macosx-x86_64:1.0.0-SNAPSHOT, org.nd4j:nd4j-native:jar:macosx-x86_64:1.0.0-SNAPSHOT: Could not find artifact org.nd4j:nd4j-native-preset:jar:macosx-x86_64:1.0.0-SNAPSHOT in snapshots-repo (https://oss.sonatype.org/content/repositories/snapshots)
```

**Comment by agibsonccc** [comment]
@b005t3r  sorry the mac GH actions workers are a bit flaky. Let me try again tomorrow. I'm  simultaneously working on publishing a staging repo for internal testing so that's clashing with the snapshot allocation we have. Alternatively, feel free to DM me on the forums: https://community.konduit.ai and I can try the model when I get a chance. Make sure to send me a new link.

**Comment by b005t3r** [comment]
I managed to download the macOS files and run it. I think it loads up the model correctly (however with warnings, given below), but for some reason it crashes when I try to run it.

I uploaded my project with all that's needed to run it to github, so would be great if you could give it a go:
https://github.com/b005t3r/dl4j-snapshot-test

Also, it creates some weird files when run:
[ops-added-old.txt](https://github.com/eclipse/deeplearning4j/files/6356573/ops-added-old.txt)
[ops-imported-old.txt](https://github.com/eclipse/deeplearning4j/files/6356575/ops-imported-old.txt)
[ops-removed-old.txt](https://github.com/eclipse/deeplearning4j/files/6356576/ops-removed-old.txt)
[variables-added-old.txt](https://github.com/eclipse/deeplearning4j/files/6356577/variables-added-old.txt)

Console output, along with the exception:
```
Warning: Versions of org.bytedeco:javacpp:1.5.5 and org.bytedeco:opencv:4.3.0-1.5.3 do not match.
10:11:49.304 [main] INFO org.nd4j.linalg.factory.Nd4jBackend - Loaded [CpuBackend] backend
10:11:49.308 [main] ERROR org.nd4j.common.config.ND4JClassLoading - Cannot find class [org.nd4j.linalg.jblas.JblasBackend] of provided class-loader.
10:11:49.308 [main] ERROR org.nd4j.common.config.ND4JClassLoading - Cannot find class [org.canova.api.io.data.DoubleWritable] of provided class-loader.
10:11:49.309 [main] ERROR org.nd4j.common.config.ND4JClassLoading - Cannot find class [org.nd4j.linalg.jblas.JblasBackend] of provided class-loader.
10:11:49.309 [main] ERROR org.nd4j.common.config.ND4JClassLoading - Cannot find class [org.canova.api.io.data.DoubleWritable] of provided class-loader.
10:11:50.103 [main] INFO org.nd4j.nativeblas.NativeOpsHolder - Number of threads used for linear algebra: 1
10:11:50.106 [main] WARN org.nd4j.linalg.cpu.nativecpu.CpuNDArrayFactory - *********************************** CPU Feature Check Warning ***********************************
10:11:50.106 [main] WARN org.nd4j.linalg.cpu.nativecpu.CpuNDArrayFactory - Warning: Initializing ND4J with Generic x86 binary on a CPU with AVX/AVX2 support
10:11:50.106 [main] WARN org.nd4j.linalg.cpu.nativecpu.CpuNDArrayFactory - Using ND4J with AVX/AVX2 will improve performance. See deeplearning4j.org/cpu for more details
10:11:50.106 [main] WARN org.nd4j.linalg.cpu.nativecpu.CpuNDArrayFactory - Or set environment variable ND4J_IGNORE_AVX=true to suppress this warning
10:11:50.106 [main] WARN org.nd4j.linalg.cpu.nativecpu.CpuNDArrayFactory - *************************************************************************************************
10:11:50.346 [main] INFO org.nd4j.nativeblas.Nd4jBlas - Number of threads used for OpenMP BLAS: 6
10:11:50.507 [main] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Backend used: [CPU]; OS: [Mac OS X]
10:11:50.507 [main] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Cores: [12]; Memory: [8.0GB];
10:11:50.507 [main] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Blas vendor: [OPENBLAS]
10:11:50.510 [main] INFO org.nd4j.linalg.cpu.nativecpu.CpuBackend - Backend build information:
 Clang: "12.0.0 (clang-1200.0.32.29)"
STD version: 201103L
DEFAULT_ENGINE: samediff::ENGINE_CPU
HAVE_FLATBUFFERS
HAVE_OPENBLAS
10:11:51.115 [main] WARN org.nd4j.imports.converters.ImportClassMapping - Duplicate TF op mapping found for op Pow: org.nd4j.linalg.api.ops.impl.scalar.Pow vs org.nd4j.linalg.api.ops.impl.transforms.custom.Pow
10:11:51.119 [main] WARN org.nd4j.imports.converters.ImportClassMapping - Duplicate TF op mapping found for op FloorMod: org.nd4j.linalg.api.ops.impl.transforms.pairwise.arithmetic.FModOp vs org.nd4j.linalg.api.ops.impl.transforms.pairwise.arithmetic.FloorModOp
10:11:51.811 [main] INFO org.nd4j.autodiff.samediff.SameDiff - Creating new InferenceSession for thread 1
Error at [/Users/runner/work/deeplearning4j/deeplearning4j/libnd4j/include/ops/declarable/generic/shape/reshape.cpp:173:0]:
Reshape: lengths before and after reshape should match, but got 0 vs 1
10:11:52.671 [main] ERROR org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner - Failed to calculate output shapes for op reshape. Attempted to execute with 2 inputs, 0 outputs, 0 targs, 0 iargs, 0 bargs and 0 dargs. Inputs: [(LONG,[],c), (INT,[1],c)]. Input var names: [Postprocessor/BatchMultiClassNonMaxSuppression/map/while/MultiClassNonMaxSuppression/FilterGreaterThan_82/Where, Postprocessor/BatchMultiClassNonMaxSuppression/map/while/MultiClassNonMaxSuppression/FilterGreaterThan_82/Reshape/shape]. Output var names: [Postprocessor/BatchMultiClassNonMaxSuppression/map/while/MultiClassNonMaxSuppression/FilterGreaterThan_82/Reshape] - Please see above message (printed out from c++) for a possible cause of error.
Exception in thread "main" java.lang.RuntimeException: Op reshape with name Postprocessor/BatchMultiClassNonMaxSuppression/map/while/MultiClassNonMaxSuppression/FilterGreaterThan_82/Reshape failed to execute.org.nd4j.linalg.cpu.nativecpu.ops.CpuOpContext@3dc95b8b Here is the error from c++: Op validation failed
	at org.nd4j.linalg.cpu.nativecpu.ops.NativeOpExecutioner.calculateOutputShape(NativeOpExecutioner.java:1690)
	at org.nd4j.linalg.api.ops.DynamicCustomOp.calculateOutputShape(DynamicCustomOp.java:545)
	at org.nd4j.autodiff.samediff.internal.InferenceSession.getAndParameterizeOp(InferenceSession.java:876)
	at org.nd4j.autodiff.samediff.internal.InferenceSession.getAndParameterizeOp(InferenceSession.java:60)
	at org.nd4j.autodiff.samediff.internal.AbstractSession.output(AbstractSession.java:385)
	at org.nd4j.autodiff.samediff.SameDiff.directExecHelper(SameDiff.java:2579)
	at org.nd4j.autodiff.samediff.SameDiff.batchOutputHelper(SameDiff.java:2547)
	at org.nd4j.autodiff.samediff.SameDiff.output(SameDiff.java:2522)
	at org.nd4j.autodiff.samediff.config.BatchOutputConfig.output(BatchOutputConfig.java:142)
	at TestTFModel.testPoseRecognition(TestTFModel.java:47)
	at TestTFModel.main(TestTFModel.java:22)
```

PS. I registered at the site, but can't send private messages there :)

**Comment by b005t3r** [comment]
I tried running my code again with the new version (1..0.0-M1.1), but it still does not work, call stack below:

```
15:29:48.842 [Test worker] INFO org.nd4j.linalg.factory.Nd4jBackend - Loaded [CpuBackend] backend
15:29:53.088 [Test worker] INFO org.nd4j.nativeblas.NativeOpsHolder - Number of threads used for linear algebra: 1
15:29:53.091 [Test worker] INFO org.nd4j.linalg.cpu.nativecpu.CpuNDArrayFactory - Binary level Generic x86 optimization level AVX/AVX2
15:29:53.336 [Test worker] INFO org.nd4j.nativeblas.Nd4jBlas - Number of threads used for OpenMP BLAS: 6
15:29:53.500 [Test worker] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Backend used: [CPU]; OS: [Mac OS X]
15:29:53.500 [Test worker] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Cores: [12]; Memory: [0.5GB];
15:29:53.500 [Test worker] INFO org.nd4j.linalg.api.ops.executioner.DefaultOpExecutioner - Blas vendor: [OPENBLAS]
15:29:53.503 [Test worker] INFO org.nd4j.linalg.cpu.nativecpu.CpuBackend - Backend build information:
 Clang: "12.0.0 (clang-1200.0.32.29)"
STD version: 201103L
DEFAULT_ENGINE: samediff::ENGINE_CPU
HAVE_FLATBUFFERS
HAVE_OPENBLAS
15:29:54.081 [Test worker] WARN org.nd4j.imports.converters.ImportClassMapping - Duplicate TF op mapping found for op Pow: org.nd4j.linalg.api.ops.impl.scalar.Pow vs org.nd4j.linalg.api.ops.impl.transforms.custom.Pow
15:29:54.085 [Test worker] WARN org.nd4j.imports.converters.ImportClassMapping - Duplicate TF op mapping found for op FloorMod: org.nd4j.linalg.api.ops.impl.transforms.pairwise.arithmetic.FModOp vs org.nd4j.linalg.api.ops.impl.transforms.pairwise.arithmetic.FloorModOp

Cannot get array for ARRAY type SDVariable - use SDVariable.exec or SameDiff.output instead
java.lang.UnsupportedOperationException: Cannot get array for ARRAY type SDVariable - use SDVariable.exec or SameDiff.output instead
	at org.nd4j.autodiff.samediff.SameDiff.getArrForVarName(SameDiff.java:747)
	at org.nd4j.imports.graphmapper.tf.TFGraphMapper.initFunctionFromProperties(TFGraphMapper.java:826)
	at org.nd4j.linalg.api.ops.impl.shape.Gather.initFromTensorFlow(Gather.java:99)
	at org.nd4j.imports.graphmapper.tf.TFGraphMapper.importGraph(TFGraphMapper.java:356)
	at org.nd4j.imports.graphmapper.tf.TFGraphMapper.importGraph(TFGraphMapper.java:141)
	at org.nd4j.imports.graphmapper.tf.TFGraphMapper.importGraph(TFGraphMapper.java:87)
	at org.nd4j.imports.graphmapper.tf.TFGraphMapper.importGraph(TFGraphMapper.java:73)
	at org.nd4j.autodiff.samediff.SameDiff.importFrozenTF(SameDiff.java:5751)
	at com.ansharstudios.shoottracer.test.TensorFlowWithDL4JTest.testPoseRecognition(TensorFlowWithDL4JTest.java:50)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:119)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:414)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

**Comment by agibsonccc** [comment]
@b005t3r just letting you know I saw this. Let me take a look when I get some bandwidth. I'll revert either tomorrow or early next week. Thanks!

**Comment by b005t3r** [comment]
@agibsonccc just wanted to ask if you maybe found some time to look into this again?

**Comment by agibsonccc** [comment]
Closing — this issue references architecture (old UI, M1/M2.1 release, Spark, TF import, ParallelWrapper training, or legacy subsystem) that has been fundamentally rewritten in the upcoming release. If you still experience a similar issue with the new version, please open a new issue with reproduction steps.
